src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -65,6 +65,7 @@ public void toStringMethod() {
                 + editPersonDescriptor.getPhone().orElse(null) + ", email="
                 + editPersonDescriptor.getEmail().orElse(null) + ", address="
                 + editPersonDescriptor.getAddress().orElse(null) + ", tags="
+                + editPersonDescriptor.getRemark().orElse(null) + ", remark="
```

Comment:
The order of printing is wrong, i.e. remarks will get printed under tags and tags will be under remarks